pkgs/applications/misc/opentrack/default.nix

```diff
@@ -0,0 +1,61 @@
+{ mkDerivation, lib, fetchFromGitHub, stdenv, cmake, pkg-config, ninja, qtbase
+, qttools, opencv4, procps, eigen, libXdmcp, libevdev, wineFull, fetchzip
+, makeDesktopItem, fetchurl, copyDesktopItems }:
+
+let
+  version = "2.3.13";
+
+  aruco =
+    import ./aruco.nix { inherit lib stdenv cmake fetchFromGitHub opencv4; };
```

Comment:
```suggestion
  aruco = import ./aruco.nix { };
```

Reply:
Happy to apply the remaining suggestions, but I don't understand how this one can work. Could you please clarify?

Reply:
If you don't list the attributes all required should be used IIRC.

Reply:
That would be with `callPackage`. I changed it to that.